pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -4,25 +4,141 @@
 package tpmmgr
 
 import (
+	"crypto/x509"
 	"flag"
 	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
 	"github.com/lf-edge/eve/pkg/pillar/agentlog"
 	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"time"
 )
 
 const (
+	//TpmPubKeyName is the file to store TPM public key file
+	TpmPubKeyName = "/config/tpm.eccpubk.der"
+
```

Comment:
Why is this in /config? The way to think about /config vs. persist is that if you remove /persist and re-create it -- the device should continue functioning, so any information that needs to exist for device to talk to the cloud for the first time (to re-create /persist) needs to go into /config.

Could you please explain what happens if /config/tpm.eccpubk.der gets removed?

Reply:
Actually, /config/tpm.eccpubk.der is temporary file, we should clean it up after generating device certificate. I will make the code change.

Reply:
This has been cleaned up, as said in my previous comment.

Reply:
I still see it above -- can you please let me know what has been cleaned up?

Reply:
Addressed. This file is not in /config anymore. It is a short-lived file, and lives in /var/tmp/